xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-notifiers/xwiki-platform-notifications-notifiers-api/src/main/java/org/xwiki/notifications/notifiers/internal/email/live/PrefilteringLiveNotificationEmailDispatcher.java

```diff
@@ -164,6 +164,7 @@ private void prepare(QueueEntry entry, Map<DocumentReference, CompositeEvent> ev
         entry.entities.forEach(entity -> {
             CompositeEvent composite = eventsToSend.get(entity);
 
+            // FIXME: This should use the grouping strategy
```

Comment:
Why not now ?

Reply:
I'm planning to work on it in a next batch, but I'm not sure to have time to do it for 15.5RC1: live email notif has its own mechanism and it might involve some more work.